src/test/resources/com/atomist/rug/test/gherkin/handler/event/PassingFeature1Steps.ts

```diff
@@ -7,6 +7,6 @@ When("a visionary leader enters", world => {
    world.registerHandler("ReturnsEmptyPlanEventHandler")
    world.sendEvent(new node.Commit)
 })
-Then("excitement ensues", (p,world) => {
+Then("excitement ensues", world => {
```

Comment:
It looks like `plan` is implemented on `HandlerScenarioWorld`, so perhaps it should remain `(p, world)`.